### ISSUE TITLE
Add ability to start game in browser tab rather than full-screen mode

### ISSUE THREAD

**Issue by bjbest60** [ainslec/adventuron-issue-tracker#279]
It'd be cool if there were a way to make Adventuron not start in full-screen mode, but rather start as a standard browser tab.  When I'm working in the editor and I split the screen between the code and the game, the physical space of the game on the screen is comfortable.  But when I run the compiled game, the size is huge and I immediately close out of full-screen mode.  I know there's a virtually infinite range of screen setups out there, but automatically starting in full-screen mode feels weird to me.

**Comment by ainslec** [comment]
I agree that this is an issue, but you should make sure that the screen {} section in your theme will make your game playable when it is full screeen. That is the purpose of the section.

Please supply some screenshots of how your game looks when full screen, and I might be able to make some scaling suggestions.

With regards to auto fullscreen, I'll see if I can add an option to enable or disable this on a per-game basis.
